actions/src/dispatcher.js

```diff
@@ -27,5 +28,12 @@ const ACTIONS = {
 module.exports = {
   async dispatch(event, action) {
     core.info(`Received Event:${event} Action:${action}.`);
+    if (event === EVENTS.ISSUES) {
+      switch (action) {
+        case ACTIONS.LABELLED:
+          await issueLabelsModule.checkLabels();
+          break;
```

Comment:
Can we do this similar to how we handle oppiabot based checks -- i.e. check for repo first and based on that trigger the test (maintain a checklist in `constants.js`)?

Reply:
Done.